doc/tutorials/specification.md

```diff
@@ -63,6 +63,9 @@ You will also be using the profile `current`,
 you can find out more about profiles in
 [the documentation](https://www.libelektra.org/plugins/profile) as well.
 
+We will be writing values to the `user:/` namespace. If you want to learn more about namespaces in general, refer to the
```

Comment:
While the sentence is not wrong, it might be misleading. E.g. `spec-mount` writes to spec:/

```suggestion
We will be writing values mostly to the `user:/` namespace. If you want to learn more about namespaces in general, refer to the
```